### ISSUE TITLE
Unable to satisfy requirements error when there is a perfect match

### ISSUE THREAD

**Issue by cscalfani** [gdotdesign/elm-github-install#27]
I'm getting the following error:

```
Resolving packages...
Solving dependencies...
  ▶ No solution found: Unable to satisfy the following requirements:

- `elm-community/maybe-extra (>= 4.0.0)` required by `elm-slate/api-2.0.0`
- `elm-community/maybe-extra (< 5.0.0)` required by `elm-slate/api-2.0.0`
- `elm-community/maybe-extra (>= 4.0.0)` required by `guardian/session-api-common-1.0.0`
- `elm-community/maybe-extra (< 5.0.0)` required by `guardian/session-api-common-1.0.0`
```

Clearly, these match.

`elm-slate/api` comes from Github and `guardian/session-api-common` comes from our in-house Gitlab server.

When I EXPLICITLY add `elm-community/maybe-extra` to my `elm-package.json`, it works.

Here's my `elm-package.json` that fails:

```
{
    "version": "1.0.0",
    "summary": "Session Api for Clients",
    "repository": "https://github.com/guardian/session-api-client.git",
    "license": "Unlicense",
    "source-directories": [
        "src"
    ],
    "exposed-modules": [
        "Api.Session.Api"
    ],
    "dependencies": {
        "elm-lang/core": "5.0.0 <= v < 6.0.0",
        "mgold/elm-random-pcg": "4.0.2 <= v < 5.0.0",
        "panosoft/elm-string-utils": "2.0.0 <= v < 3.0.0",
        "panosoft/elm-utils": "2.0.0 <= v < 3.0.0",
        "panosoft/elm-cmd-retry": "3.0.0 <= v < 4.0.0",
        "panosoft/elm-parent-child-update": "2.0.0 <= v < 3.0.0",
        "elm-slate/command": "1.0.0 <= v < 2.0.0",
        "elm-slate/api": "2.0.0 <= v < 3.0.0",
        "elm-slate/common": "2.0.0 <= v < 3.0.0",
        "elm-slate/query": "2.0.0 <= v < 3.0.0",
        "guardian/entities": "1.0.0 <= v < 2.0.0",
        "guardian/session-api-common": "1.0.0 <= v < 2.0.0",
        "guardian/remote-api-common": "1.0.0 <= v < 2.0.0",
        "guardian/remote-api-client": "1.0.0 <= v < 2.0.0",
        "guardian/session-service-common": "1.0.0 <= v < 2.0.0"
    },
    "dependency-sources": {
        "guardian/entities": "git@gitlab.panosoft.com:guardian/entities",
        "guardian/session-api-common": "git@gitlab.panosoft.com:guardian/session-api-common",
        "guardian/remote-api-common": "git@gitlab.panosoft.com:guardian/remote-api-common",
        "guardian/remote-api-client": "git@gitlab.panosoft.com:guardian/remote-api-client",
        "guardian/session-service-common": "git@gitlab.panosoft.com:guardian/session-service-common"
    },
    "elm-version": "0.18.0 <= v < 0.19.0"
}

```

**Comment by gdotdesign** [comment]
This commit should fix this issue: https://github.com/gdotdesign/elm-github-install/commit/16c50824d591fdb533fba389745204cba23e1902 I'll let you know once it's out.

**Comment by gdotdesign** [comment]
This is fixed in v1.3.0.
